client/src/components/scheduler/RescheduleCancelAppointment.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import {
+  Typography,
+  Grid,
+  makeStyles,
+  TextField,
+  Button,
+} from "@material-ui/core";
+import { Link } from "react-router-dom";
+import axios from "axios";
+
+const RescheduleCancelAppointment = ({ variant, appointmentId, eventUrl }) => {
+  const reschedule = variant === "reschedule";
+  const cancel = variant === "cancel";
+  const classes = useStyles();
+  const cancelAppointment = (event) => {
+    axios.delete(`/api/appointment/cancel/${appointmentId}`).then(() => {
+      setAppointmentCancelled(true);
+    });
+  };
+
+  const linkToReschedule = `/appt/${eventUrl}`;
```

Comment:
Should the host url be included in here?

Reply:
Since I'm reusing the scheduler component, the url is required. After clicking on the reschedule button, the frontend sends a request to the backend to cancel the appointment, and redirects the user to the scheduling page, which requires the above link to work (not the most elegant solution but Sina and Nellie suggested to go with the simple way for now.)